test/api/run

```diff
@@ -99,6 +99,15 @@ query_api() {
     echo "$out"
 }
 
+# Test that scyla is serving the api port.
+check_api() {
+    out=$(query_api "$@")
+    if echo "$out" | grep -q '"code"'; then
```

Comment:
That's a strange idiom for matching in shell (why not use a "case", without echo or grep?) 

By the way, am I right that this is just a very basic check for the API. It just checks that the API doesn't result in an error, but nothing about what it does. Right?

Reply:
> That's a strange idiom for matching in shell (why not use a "case", without echo or grep?)

yeah, case would be good here.

> By the way, am I right that this is just a very basic check for the API. It just checks that the API doesn't result in an error, but nothing about what it does. Right?

correct